search/search_quality/assessment_tool/main_view.cpp

```diff
@@ -43,6 +44,14 @@ MainView::MainView(Framework & framework) : m_framework(framework)
   InitMapWidget();
   InitDocks();
   InitMenuBar();
+
+  m_framework.SetMapSelectionListeners(
+      [this](place_page::Info const & info) {
```

Comment:
I'm suspecting that we should stop the process of improving readability for this piece of code, as, from my point of view, it's readable enough.